### DISCUSSION TITLE
Rocket v0.5.0

### DISCUSSION THREAD

**Discussion by SergioBenitez** [rwf2/Rocket#2645] [Announcements]
See the [CHANGELOG](https://github.com/SergioBenitez/Rocket/blob/v0.5.0/CHANGELOG.md), [release news article](https://rocket.rs/v0.5/news/2023-11-17-version-0.5/), [RWF2 announcement](https://rocket.rs/v0.5/news/2023-11-17-rwf2-prelaunch/), [migration guide](https://rocket.rs/v0.5/guide/upgrading/), and [FAQ](https://rocket.rs/v0.5/guide/faq/) for more information.

<hr /><em>This discussion was created from the release <a href='https://github.com/SergioBenitez/Rocket/releases/tag/v0.5.0'>Rocket v0.5.0</a>.</em>

**Comment by RenaudDenis** [comment]
Well done, Mr. Benitez.
This release looks amazing.
This renewed commitment to maintaining and evolving Rocket will hopefully have a positive effect on Rocket's adoption.

**Comment by donnyyang123** [comment]
I have been using the RC version with good performance stability, and it has been used in basic services such as ETL, verification codes, and payments. Thank you SergioBenitez

**Comment by zhx09172** [comment]
It‘s great,thank u sir!
